docs/sources/enterprise/http_api.md

```diff
@@ -0,0 +1,72 @@
++++
+title = "Enterprise HTTP APIs"
+description = "Grafana Enterprise APIs"
+keywords = ["grafana", "enterprise", "api"]
+aliases = ["/docs/grafana/latest/enterprise/http_api/"]
+type = "docs"
+[menu.docs]
+name = "HTTP APIs"
+parent = "enterprise"
+weight = 1000
++++
+
+# Enterprise HTTP APIs
+
+## Authentication
+
+To access the API you need to authenticate yourself. For more information about API authentication, refer to [Authentication API]({{< relref "../http_api/auth.md" >}}).
+
+## Reporting API
+
```

Comment:
Need an intro here. Something like

This API allows you to interact programmatically with the [Reporting](LINK) feature.